components/yaml.rst

```diff
@@ -115,7 +115,7 @@ contents of the given file path and converts them to a PHP value::
 
     use Symfony\Component\Yaml\Yaml;
 
-    $value = Yaml::parseFile('/path/to/file.yaml');
+    $value = Yaml::parseFile(file_get_contents('/path/to/file.yaml'));
```

Comment:
The reason for having the `parseFile()` method is precisely not having to read the file contents before. Do you maybe confuse this method with `parse()`?

Reply:
Sorry, not awake. Thanks @xabbuh